### PR TITLE
Cost Management Jobs

### PR DESCRIPTION
**Pull request by fdevans**

### CODE REVIEW

**Comment by jsboak** [comment]
it's nice to include the specific IAM permissions so that customers know which permissions to add to the IAM role:
- *Identify unused Security Groups*:
```		
{
    "Sid": "VisualEditor0",
    "Effect": "Allow",
    "Action": "ec2:DescribeNetworkInterfaces",
    "Resource": "*"
}
```
- *Identify Unused Lambda*:
```
{
    "Sid": "VisualEditor0",
    "Effect": "Allow",
    "Action": [
    "lambda:ListFunctions",
    "logs:DescribeLogGroups",
    "logs:DeleteLogGroup",
    "lambda:DeleteFunction"
    ],
    "Resource": "*"
}
```
- The *Identify Unused VPCs* has incomplete prerequisites section (it doesn't specify the permissions needed)

File: AWS/AWS_-_Identify_Unused_VPCs.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
+- defaultTab: output
+  description: |
+    Generate list of unused Virtual Private Networks
+
+    ### Description:
+    The automation job is designed to identify VPCs that are unused across mulitiple resource types.  They would be candidates for removal.
+
+    ### Prerequisites:
+    - AWS-CLI installed on runner.
+    - API permissions for
```
Comment (jsboak) [inline AWS/AWS_-_Identify_Unused_VPCs.yaml:10]: need permissions here

Comment (fdevans) [reply AWS/AWS_-_Identify_Unused_VPCs.yaml:10]: Pushed a new file generated based on the other doc pages

**Review by jsboak — APPROVED** [review]
LGTM

**Pull request merged**